Detanup01/gbe_fork
# Add some missing implementations

* Add missing implementation for these undocumented interfaces:
  - `ISteamNetworkingSockets010`
  - `ISteamNetworkingSockets011`
  - `ISteamRemoteStorage015`
  - `ISteamUGC011`
* Proper handling for `SteamNetworkingSockets007`, real client returns a null pointer
* Set the exit code in the client API `CreateInterface()`
* Remove unused helper functions
* Dev note on how to add orphan branch as submodule

The interfaces were based on reversing the vftable from the real steamclient64.dll
1. Download any free game
2. Use the emu to locate the call to CreateGenericInterface()
3. Restore original files, change the requested interface name/version to whatever you like, then get the pointer from real client
4. Using the dev guide `how different versions of a steam interface are dispatched`, check each function from the vftable, luckily Valve has a debug log entry for the function name so you can confirm easily if you're on the right track
